addons/hw_drivers/tools/helpers.py

```diff
@@ -330,3 +330,19 @@ def write_file(filename, text, mode='w'):
         path = path_file(filename)
         with open(path, mode) as f:
             f.write(text)
+
+def download_from_url(download_url, path_to_filename):
+    try:
+        urllib3.request.urlretrieve(download_url, path_to_filename)
```

Comment:
Not very fan of creating an helper to download any file anywhere you want. If you are only using it once in one module in enterprise, keep it there first. Also don't forget to add timeout when downloading files.

Reply:
Actually for now I am only using it here but I am currently working on another PR which will use the same thing (download + unzip a different dynamic library used by another terminal) so long term it is useful to have a helper for it.